manifests/repo/yum.pp

```diff
@@ -0,0 +1,25 @@
+# Configure yumrepo
+#
+class fluentd::repo::yum (
+  $descr    = 'TreasureData',
+  $baseurl  = 'https://packages.treasuredata.com/2/redhat/$releasever/$basearch',
+  $enabled  = '1',
+  $gpgcheck = '1',
+  $gpgkey   = 'https://packages.treasuredata.com/GPG-KEY-td-agent',
+) {
+
+  include '::yumrepo'
```

Comment:
is this really needed? 
yumrepo is one of puppets built-in types and doesn't need an extra module afaik